Kickstarter-iOS/Features/Activities/Datasource/ActivitiesDataSource.swift

```diff
@@ -7,39 +7,9 @@ internal final class ActivitiesDataSource: ValueCellDataSource {
   internal enum Section: Int {
     case erroredBackings
     case surveys
-    case facebookConnect
-    case findFriends
     case activities
   }
 
-  internal func facebookConnect(source: FriendsSource, visible: Bool) {
-    self.set(
-      values: visible ? [source] : [],
-      cellClass: FindFriendsFacebookConnectCell.self,
```

Comment:
No, this one is being used elsewhere. It's likely code that we're not actually using, but I'd need to dig through discover and settings to confirm, so that seems out of scope for this pr.